docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,128 @@
-
-"dialog" if present must be a boolean. By default, it's set to true. If enabled, [events](#events) are turned off as the updater handles everything. If you need the custom events, you MUST turn off the built-in dialog.
-
-"pubkey" must be a valid public-key generated with Tauri CLI. See [Signing updates](#signing-updates).
+import Command from '@theme/Command'
 
-### Update Requests
+# Updater
 
-Tauri is indifferent to the request the client application provides for update checking.
+Tauri offers a built-in updater for the MSI (Windows), AppImage (Linux) and App bundle (macOS) distribution formats.
 
-`Accept: application/json` is added to the request headers because Tauri is responsible for parsing the response.
+Once your Tauri project is ready, you can configure Tauri's updater to enable auto updating for your users.
 
-For the requirements imposed on the responses and the body format of an update, response see [Server Support](#server-support).
+## Signing updates
 
-Your update request must _at least_ include a version identifier so that the server can determine whether an update for this specific version is required.
+Tauri's updater has a built-in signature to ensure updates are safe to be installed. This safety measurement can not be turned off.
 
-It may also include other identifying criteria, such as operating system version, to allow the server to deliver as fine-grained an update as you would like.
+To sign your updates you need two things:
 
-How you include the version identifier, or other criteria is specific to the server from which you request updates. A common approach is to use query parameters, [Configuration](#configuration) shows an example.
+- The _public key_, which will be added to your `tauri.conf.json` file later, to validate update artifacts before the installation.
+- The _private key_, which is used to sign your update artifacts and should NEVER be shared with anyone. Also, if you loose this key, you will NOT be able to publish new updates to your current user base. It is crucial to store it in a safe place where you can always access it.
 
-### Built-in dialog
+To generate the keys, you need to use the Tauri CLI:
 
-By default, the updater uses a built-in dialog API from Tauri.
+<Command name="signer generate -w ~/.tauri/myapp.key" />
 
-![New Update](https://i.imgur.com/UMilB5A.png)
+If you are on Windows, you should use `$HOME/.tauri/myapp.key` instead.
 
-The dialog release notes are represented by the update `note` provided by the [server](#server-support).
-If the user accepts, the update is downloaded and installed. Afterward, the user is prompted to restart the application.
+## Tauri configuration
 
-### Javascript API
-
-:::caution
-You need to _disable built-in dialog_ in your [tauri configuration](#configuration); Otherwise, the javascript API will NOT work.
-:::
+Now you need to configure Tauri's updater. To do this, add this to your Tauri config file:
```

Comment:
```suggestion
Now you need to configure Tauri's updater. To do this, add this to your Tauri config:
```
I think I'd add some links to the config API docs here (for now just the main page maybe?)